zkg

```diff
@@ -571,22 +571,22 @@ def cmd_test(manager, args, config, configfile):
             print_error(
                 str.format('error: failed to run tests for "{}": {}', name, error_msg)
```

Comment:
It'd be consistent to add the version here and above in line 562 as well.

Reply:
done